CHANGELOG.md

```diff
@@ -30,6 +30,77 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v9.0.2](https://github.com/shopsys/shopsys/compare/v9.0.1...v9.0.2) - 2020-08-27
+
+### [shopsys/framework]
+
+#### Changed
+
+- [#1750] Product::setProductCategoryDomains() fix
+- [#1941] better extensibility of article form type
+- [#1972] fixed removing friendly urls
+- [#1986] design - admin - title button wrap bar only if contains buttons
+- [#1987] design - admin - domain switcher for more domains
+- [#1988] fixed stylelint phing targets
+
+#### Fixed
+
+- [#1823] remove final from relevant classes, Thanks to [@pk16011990]
+- [#1932] updated AdministratorFrontSecurityFacade::isAdministratorLogged() to work well if used on frontend, Thanks to [@pk16011990]
+- [#1956] IndexExportedEvent is called after elasticsearch export cron module finished
+- [#1963] fixed multiple binding of product picker events
+- [#1966] fixed clearing form options in product picker
+- [#1967] select distinct product for vat replace, Thanks to [@mposchl]
+- [#1971] fixed joining the same table with different alias
+- [#1977] remove customer and his addresses with customer user
+- [#1979] spinbox plus now works correctly with min value
+- [#1982] categories in administration are now loaded using admin locale
+- [#1998] fixed delayed javascript validators
+- [#2001] AbstractFileUploadType: remove fileConstraints class property
+- [#2004] remove space in error page filename, Thanks to [@pk16011990]
+- [#2006] sellingTo is now set with the end of day time
```

Comment:
changed

Reply:
This is fix to me. The whole time it was inserting the wrong datetime.